File-Formats/DocIO/Convert-Word-Document-to-PDF-in-AWS-Lambda.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Convert Word to PDF in Amazon Web Services on Lambda | Syncfusion
+description: Convert Word document to PDF in  Amazon Web Services on Lambda using .NET Core Word (DocIO) and  PDF library, without Microsoft Word or interop dependencies.
+platform: file-formats
+control: DocIO
+documentation: UG
+---
+
+# Convert Word Document to PDF in Amazon Web Services on Lambda
+
+Syncfusion  DocIO is a [.NET Core Word library](https://www.syncfusion.com/document-processing/word-framework/net-core/word-library) used to create, read, edit and **convert Word documents** programmatically without **Microsoft Word** or interop dependencies. Using this library, you can **convert a Word document to PDF in Amazon Web Services on Lambda**
+
+## Steps to convert Word document to PDF in Amazon Web Services on Lambda
```

Comment:
Bold the options as per KB.
Remove unwanted empty lines.

Reply:
updated